js/tji-datasets-explore.js

```diff
@@ -476,9 +478,12 @@ TJIChartView.prototype.create_filters = function() {
 
 TJIChartView.prototype.create_filter_checkboxes = function(filter) {
   var that = this;
-  var fieldset = jQuery('<fieldset><legend>' + filter.name.replace(/_/g, " ") + '<i class="fas fa-caret-down"></i></legend></fieldset>');
+  var fieldset = jQuery('<fieldset><legend class="js-chart-legend">' + filter.name.replace(/_/g, " ") + '</legend></fieldset>');
+  var filterset = jQuery('<div class="js-filter-set"><a class="js-toggle-select">Select All</a> / <a class="js-toggle-unselect">Unselect All</a></div>');
```

Comment:
Same with this, the select-all and deselect-all hasn't yet been merged, so I'm not sure where this code is coming from?